core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -70,14 +70,24 @@
     </li>
 
     {% if username %}
-      <li ng-if="isLargeScreen" class="dropdown" popover-is-open="postTutorialHelpPopoverIsShown"
+    <div class="oppia-help-dropdown nav navbar-nav oppia-navbar-nav pull-right ng-cloak">
```

Comment:
It gives error if adopt so;
<li class="oppia-help-dropdown dropdown" popover-is-open="postTutorialHelpPopoverIsShown" popover-placement="bottom" popover-trigger="none" popover="To get help in the future, click here!">
As popover is inline with the `<li>`tag itself